tests/integration/downstream/downstream_test_case.py

```diff
@@ -0,0 +1,38 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+
+from app.submitter.submitter import SubmitterFactory, Submitter
+from app.submitter.converter import Converter
+
+
+class MyMockSubmitter(Submitter):
+    def __init__(self):
+        self._message = None
+        self._submitted_at = None
+
+    def send_answers(self, user, metadata, schema, answers):
+        message, submitted_at = Converter.prepare_answers(user, metadata, schema, answers)
```

Comment:
I still think it would of been better to refactor the submitter, extract the encrypting functionality into a separate method and then override that and the send message method instead. This just means that if the send_answers method or the Converter.prepare_answer method signature changes we've got to remember to change it here too.